sim/common/vanilla/item_effects.go

```diff
@@ -252,28 +254,63 @@ func init() {
 			},
 		})
 
-		fistOfShahramAuras := character.NewPartyAuraArray(func(unit *core.Unit) *core.Aura {
+		/*fistOfShahramAuras := character.NewPartyAuraArray(func(unit *core.Unit) *core.Aura {
 			return unit.GetOrRegisterAura(core.Aura{
 				ActionID: core.ActionID{SpellID: 16601},
 				Label:    "Fist of Shahram",
 				Duration: time.Second * 8,
+				MaxStacks: 5,
 				OnGain: func(aura *core.Aura, sim *core.Simulation) {
 					character.MultiplyAttackSpeed(sim, 1.3)
 				},
 				OnExpire: func(aura *core.Aura, sim *core.Simulation) {
 					character.MultiplyAttackSpeed(sim, 1/1.3)
 				},
 			})
-		})
+		})*/
```

Comment:
Could you clean up these old auras if they're no longer needed?

Reply:
cleaned up